pytensor/tensor/blas.py

```diff
@@ -1179,7 +1127,7 @@ def c_code(self, node, name, inp, out, sub):
     def c_code_cache_version(self):
         gv = self.build_gemm_version()
         if gv:
-            return (7,) + gv
+            return (6,) + gv
```

Comment:
We should increase the number. Next time we change the code we won't remember we should jump two numbers

Reply:
```suggestion
            return (8,) + gv
```